helm/hpcc/templates/_helpers.tpl

```diff
@@ -1370,6 +1376,109 @@ spec:
 {{- end }}
 {{- end }}
 
+{{/*
+Builds the commonName for a client certificate.  Used in creation of both certificate and access control list.
```

Comment:
useful in keeping with [most] other helpers, if comment says what helper template expects to be passed in in dictionary, e.g. "Pass in root, me .. "